src/style.css

```diff
@@ -2,0 +1,91 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir.ttc);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: AvenirBook;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: AvenirHeavy;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+body {
+  margin: 0;
+}
+
+.container {
+  margin: 20px 8px 0;
+}
+
+.search__bar {
+  box-sizing: border-box;
+  width: calc(100% - 16px);
+  font-family: Avenir, sans-serif;
+  color: #3d4e61;
+  border: 1px solid #e1e7ed;
+}
+
+.search__bar::placeholder {
+  color: #3d4e61;
+}
+
+.search__bar--big {
+  height: 70px;
+  padding-left: 62px;
+  font-size: 16px;
+  line-height: 21.86px;
+  border-radius: 3px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+}
+
+.search__bar--big:hover {
+  box-shadow: 0 3px 8px 0 rgba(61, 78, 97, 0.2);
+}
+
+.search__bar--big:focus {
+  font-family: AvenirHeavy, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. The `.search__bar--big:focus` style specifies `AvenirHeavy` without a generic family as a fallback. Add a generic font family like `sans-serif` to ensure text is displayed with a reasonable default if the custom font fails to load.